runtime/src/iree-amd-aie/aie_runtime/iree_aie_runtime.cc

```diff
@@ -251,7 +232,7 @@ uint32_t AMDAIEDeviceModel::getNumBDs(uint8_t col, uint8_t row) const {
   AMDAIETileType tileType = getTileType(col, row);
   if (tileType == AMDAIETileType::SHIMPL || tileType == AMDAIETileType::MAX)
     return 0;
-  const XAie_DmaMod *dmaMod =
+  const XAie_DmaMod* dmaMod =
```

Comment:
Where are these style changes coming from? For me, `clang-format` formats it like it was before.

Reply:
Possibly my editor is misconfigured on the new laptop

Reply:
Ran it by hand just now and it didn't restore things :shrug:

Reply:
I guess different versions/settings? We should choose something to avoid local formatters changing things back and forth.

Reply:
I would keep it like it was before as that's the current format used everywhere else.

Reply:
Which version are you using?

Reply:
I tried 16 and 17 and no change

Reply:
14 it looks like

Reply:
Just running `clang-format-14` didn't do it but `clang-format --style=LLVM` and then `clang-format --style=Google` did it. Of course it also made changes to `iree_aie_runtime.h` where there hadn't been :shrug:. But yea we should just have formatting checked in CI.